_posts/2022-04-07-launching-open-source-operator-for-openSearch.markdown

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "Introducing Kubernetes Open-Source Operator for OpenSearch"
+authors:
+  - Opster
+date: 2022-04-07
+categories:
+  - technical-post
+twittercard:
+  description: "The Kubernetes OpenSearch Operator is used for automating the deployment, provisioning, management, and orchestration of OpenSearch clusters."
+---
+
+
+Launching Open-Source Operator for OpenSearch 
+----------------------------------------------
+
+### With powerful capabilities for orchestration & management of OpenSearch clusters. The Operator is led by Opster with partners including AWS, SUSE Rancher, Maibornwolff, Logz.io and more.
+
+Overview
+--------
+
+The [OpenSearch Operator](https://github.com/Opster/opensearch-k8s-operator) is fully open-source, licensed as Apache 2.0, and is used for automating the deployment, provisioning, management, and orchestration of OpenSearch clusters and OpenSearch dashboards.
+
+The Operator development is being led by [Opster](https://opster.com/) with partners including [SUSE Rancher](https://www.suse.com/), [Maibornwolff](https://www.maibornwolff.de/en), [AWS](https://aws.amazon.com/), [Logz.io](https://logz.io/) and more.
+
+Operator Capabilities
+---------------------
+
+The Operator enables high-level API use, for easily running advanced OpenSearch operations on Kubernetes. With the Operator, scaling up and down, version upgrades, rolling restarts, securing deployments and managing certificates is simplified and streamlined.
+
+The Operator allows for management of multiple OpenSearch clusters and OpenSearch Dashboards. The Operator simplifies operation by providing node draining, adjustment of memory and disk resources on the nodes, advanced shard allocation strategies and auto-scaling based on usage load and resources.
```

Comment:
Some of these features (e.g. advanced shard allocation strategies, autoscaling based on usage load and resources) are not supported currently (reference - [README](https://github.com/Opster/opensearch-k8s-operator/blob/main/README.md)). This can cause confusion. When talking about capabilities, it would probably be a better idea to highlight what's supported (in this release) and what's upcoming separately to avoid confusion.